app/views/meeting.status.html

```diff
@@ -3,21 +3,53 @@
     <div class="ad-page-header">
         <div class="container col-xs-12">
             <div class="row">
-                <h3 ng-if="msc.mobile" class="col-xs-10">{{msc.title}}</h3>
-                <i ng-if="menuFull()" class="col-xs-1 col-sm-3 glyphicon glyphicon-info-sign left" aria-hidden="true" ng-click="msc.showInfo()"></i>
-                <i ng-if="menuFull()" ng-if="!msc.mobile" class="col-sm-3 glyphicon glyphicon-home left" aria-hidden="true" ng-click="msc.goHome()"></i>
-                <i ng-if="!msc.mobile" class="col-sm-3 glyphicon glyphicon-menu-left left" aria-hidden="true" ng-click="closeMenu()"></i>
-                <i ng-if="!msc.mobile" class="col-sm-3 glyphicon glyphicon-menu-right left" aria-hidden="true" ng-click="openMenu()"></i>
+                <i ng-if="menuFull()" class="col-xs-1 col-sm-3 col-md-2 col-lg-3 glyphicon glyphicon-info-sign left" aria-hidden="true" ng-click="msc.showInfo()"></i>
+                <i ng-if="menuFull()" class="col-sm-3 col-md-2 col-lg-3 glyphicon glyphicon-home left" aria-hidden="true" ng-click="msc.goHome()"></i>
+                <i class="col-sm-3 col-md-2 col-lg-3 xs-hidden glyphicon glyphicon-menu-left left" aria-hidden="true" ng-click="closeMenu()"></i>
+                <i class="col-sm-3 col-md-2 col-lg-3 xs-hidden glyphicon glyphicon-menu-right left" aria-hidden="true" ng-click="openMenu()"></i>
             </div>
         </div>
     </div>
 
     <div class="ad-page-content">
 
-        <button type="button" class="ng-class: ['btn', 'btn-primary', 'btn-lg']" ng-click="">{{msc.object.meetingTitle}} <span class="badge">8</span>
-        </button>
-        <button type="button" class="btn btn-primary btn-lg" ng-click="">Kokousstatus <span class="badge">9</span>
-        </button>
+        <button type="button" class="btn btn-primary btn-lg disabled ad-dummy-button" ng-click="">{{msc.object.meetingTitle}}</button>
+
+        <div class="btn-group btn-block" uib-dropdown is-open="">
+            <button id="statusButton" type="button" class="btn btn-primary btn-lg ad-button" uib-dropdown-toggle>
+                <div>{{msc.object.meetingStatus}} {{msc.object.topicList.length}} Status</div>
+                <span class="caret"></span>
+            </button>
+            <ul uib-dropdown-menu role="menu" aria-labelledby="statusButton">
```

Comment:
Let's not implement the status changing dropdowns at this phase, only later when extended to cover the specific user role.